README.md

```diff
@@ -86,10 +86,23 @@ Important typedefs required for parsing ONNX models are declared in this header:
 
 ### Docker image
 
-Build the onnx_tensorrt Docker image by running:
+#### Tar-Based TensorRT
 
-    cp /path/to/TensorRT-6.0.*.tar.gz .
-    docker build -t onnx_tensorrt .
+Build the onnx_tensorrt Docker image using tar-based TensorRT by running:
+
+    git clone --recurse-submodules https://github.com/onnx/onnx-tensorrt.git
+    cd onnx-tensorrt
+    cp /path/to/TensorRT-6.0.*.tar.gz /docker
```

Comment:
cp /path/to/TensorRT-6.0.*.tar.gz **docker/**

Reply:
Fixed.

Reply:
Fixed and tested again. Sorry about the mistake.